src/components/TimelineEvent.js

```diff
@@ -2,9 +2,19 @@ import React from 'react';
 import './TimelineEvent.css';
 import Timestamp from './Timestamp';
 
-const TimelineEvent = () => {
+const TimelineEvent = (props) => {
   // Fill in your code here
-  return;
+  return (
+    <section>
+
+    {/* <h3>{ props.timestamp } </h3> */}
+      <h3> <Timestamp time= { props.time }/></h3>
```

Comment:
This part is a little nuts, so bear with it ... Timestamp knows how to make the Very-Robotic-Official-Computer-Science time element from that the JSON has in it into a sentence like "a week ago" or whatever. App.js gave us this stupid time format that's ugly and doesn't fit, but if we REGIFT it to Timestamp, it can use it. Coooool. So the part where it's saying `Timestamp time= { props.time }`  is where TimelineEvents hands the props.time it JUST GOT from App over to Timestamp.